ordered logistic regression.ipynb

```diff
@@ -16,34 +16,50 @@
    "outputs": [],
```

Comment:
<p>I'd rather not modify the model definition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MarcoGorelli/pydataglobal-21/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I see. This seemed the right approach to me, so to facilitate the reading of the model structure. Furthermore, in this P.R., I was trying to test the OrderedLogistic approach that you mentioned during PYDATA.

Reply:
Instead of deleting the whole example of using the OrderedLogistic function (which is a major asset for didactic purposes), I added a new Jupiter-notebook file. Therefore, now, we have 2 examples: one using the original model definition that you showed on PYDATA (i.e., using the Categorical function), and another in which the OrderedLogistic is applied.

I reinforce the need of supporting the OrderedLogistic example (Jupiter-notebook) since the output of this model is completely different from the former one (i.e., the Categorical example). As a consequence, the approach required for one to plot the expected priors from the (untrained/trained) model is completely different for each use case.

Reply:
> I reinforce the need of supporting the OrderedLogistic example (Jupiter-notebook) since the output of this model is completely different from the former one (i.e., the Categorical example).

Thanks - sounds like a bug somewhere then, I'll take a look, I'd have expected them to be the same

Reply:
I wouldn't say a bug. It is just that there are some minor differences in the objects returned. Feel free to check the differences in each Jupyter-notebook.

Reply:
> Thanks for our PR @PhilipeRLeal , just made some suggestions

Dear @MarcoGorelli,

do you think that this Github Repository could be used/provided in the PYMC default examples?

I believe that the provided Jupyter-notebooks have some interesting analyses and example cases that are yet not available on PYMC's website.

Sincerely,

Reply:
Are there minor differences, or is it that

> the output of this model is completely different

? I just tried this and they look about the same.

Either way, I'd rather not refactor the model, it's fine as a single function

Reply:
> I believe that the provided Jupyter-notebooks have some interesting analyses and example cases that are yet not available on PYMC's website.

This'd be a question for the PyMC docs team, but I expect it'd be a welcome addition